lib/paypal-adaptive.js

```diff
@@ -133,9 +129,6 @@ Paypal.prototype.callApi = function (apiMethod, data, callback) {
     if (config.deviceIpAddress)
         options.headers['X-PAYPAL-DEVICE-IPADDRESS'] = config.deviceIpAddress;
 
-    if (config.subject)
-        options.headers['X-PAYPAL-SECURITY-SUBJECT'] = config.subject;
-
```

Comment:
This is necessary for some people. It was added by a PR.
Why did you remove it?